src/System.Security.Cryptography.X509Certificates/tests/CertTests.cs

```diff
@@ -10,13 +10,13 @@ namespace System.Security.Cryptography.X509Certificates.Tests
     public static class CertTests
     {
         [Fact]
-        [ActiveIssue(1977, PlatformID.Any)]
+        [ActiveIssue(1993, PlatformID.AnyUnix)]
         public static void X509CertTest()
         {
             const string CertSubject =
                 @"CN=Microsoft Corporate Root Authority, OU=ITG, O=Microsoft, L=Redmond, S=WA, C=US, E=pkit@microsoft.com";
 
-            using (X509Certificate cert = new X509Certificate("microsoft.cer"))
+            using (X509Certificate cert = new X509Certificate("TestData\\microsoft.cer"))
```

Comment:
I assume we'll eventually enable these for Unix as well?  Can we use Path.Combine for these paths instead, e.g. Path.Combine("TestData", "microsoft.cert")?